src/pybind/mgr/dashboard/frontend/applitools.config.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
   showLogs: true,
   saveDebugData: true,
   failCypressOnDiff: true,
-  concurrency: 4
+  concurrency: 4,
+  baselineBranchName: 'master'
```

Comment:
yup, i agree. I'll try to use the ceph_release file and get the value. Do I need to include `APPLITOOLS_PARENT_BRANCH_NAME ` in this file or since it is already in the ceph-build repo, shall we ignore it?

Reply:
I'd rather put everything in the same location, but the parent branch might only make sense for PRs and hence it cannot be guessed except from the Jenkins environment variables.

I was reading the applitools docs and it seems that the baseline can be inferred from the parent branch. However, since we're not setting the baseline-parent branch for `master`/`pacific`/etc, applitools doesn't seem to be able to track baselines for release branches (in the 'merge branch' page it doesn't show master/pacific, but only default).